libraries/resource_providers/api300/synergy/server_profile_provider.rb

```diff
@@ -12,15 +12,8 @@
 module OneviewCookbook
   module API300
     module Synergy
-      # ServerProfile API300 Synergy provider
-      class ServerProfileProvider < API200::ServerProfileProvider
-        def create_or_update
-          load_os_deployment_plan
-          super
-        end
-
-        protected
-
+      # Helper module containing methods for loading OS Deployment Plans into @item
+      module ServerProfileHelpers
```

Comment:
An opinion :grin: : I don't think it necessary to create a mixin. The class became more swollen. Although this doesn't bother me.

Reply:
That is used by [`libraries/resource_providers/api500/synergy/server_profile_template_provider.rb`](https://github.com/HewlettPackard/oneview-chef/pull/327/files/d4dfea08306efe66b95cc563e93252a5ef28fd1a..ef351918d508bf8cf3483e3227bc8df8366fd883#diff-9581a27e30afac21d7000b86bd4be3d6) too.

Reply:
It's ok! Sorry guys. :sweat_smile:

Reply:
😄 😄